index.js

```diff
@@ -0,0 +1,96 @@
+let firstNumber;
+let display = document.querySelector('.display')
+display.textContent="";
+
+let clear = document.querySelector('#clear')
+let operators = document.querySelectorAll('.operator')
+let operator;
+let results;
+let secondNumber;
+let getElement = document.querySelectorAll('#click')
+let operatorClicked = false;
+
+const sum = ()=>{
```

Comment:
Your operator methods perform the calculation twice. Once for the return and once to assign it to results. Later in the operate function you again assign to results, so I guess you can remove the assignments from the `sum` / `substract` / `divide` / `multiply` methods.